### PR TITLE
Skeleton for GitLab support

### PR DESCRIPTION
**Pull request by artie-shevchenko**
Re #29. "GitLab support"

GitLab repos list management is ready as well as all the structural support for GitLab implementation I think.

TODOs for GitLab support MVP:

1. Start with a `gitLabTokenForm` on `popup.html` and create an event handler for it in `popup.ts`. Make sure it's a valid token and store it by calling `storeGitLabUser`.
2. Init GitLab API client in `gitlab.ts`.
3. Implement GitLab repo sync in `gitlab.ts`, similar to `github.ts`. I think we should be able to reuse `MyPR#ofGitHubResponses` without any changes, but if we're not lucky and there are different models what are reviews then might be necessary to tweak it.
4. [p2] Add tokens management to Settings, to make it possible to have sync both with GitHub and GitLab repos.
5. [p2] Design and implement behavior in case there are problems with GitHub token, but not with GitLab.

There are TODO(29) in code, hopefully helpful.

### CODE REVIEW

**Comment by Onexx** [comment]
Thanks for this and for other improvements I missed! I made my mind to keep refactoring as small as possible. I'll try to add gitlab  support ASAP after it

**Comment by artie-shevchenko** [comment]
@Onexx this is not production-ready, I've reverted

File: src/options.ts [hunk 1]
```diff
@@ -1,23 +1,26 @@
 import "../styles/options.scss";
```
Comment (artie-shevchenko) [inline src/options.ts:1]: test comment

Comment (whatever-artie) [reply src/options.ts:1]: reply to test comment

Comment (artie-shevchenko) [reply src/options.ts:1]: reply to reply

Comment (whatever-artie) [reply src/options.ts:1]: @artie-shevchenko

**Comment by whatever-artie** [comment]
@artie-shevchenko comment on the PR itself

File: src/options.ts (hunk 1 above)
Comment (whatever-artie) [reply src/options.ts:1]: another comment

**Comment by artie-shevchenko** [comment]
test comment

**Pull request merged**